.github/workflows/test-base.yml

```
@@ -104,21 +104,14 @@ jobs:
-      run: |
-        make run-e2e-env > /tmp/backend-e2e.log 2>&1 &
-        echo $! > /tmp/backend-e2e.pid
-        sleep 5
-        # Check if backend started successfully
-        if ! kill -0 $(cat /tmp/backend-e2e.pid) 2>/dev/null; then
-          echo "Backend failed to start. Log:"
-          cat /tmp/backend-e2e.log
-          exit 1
-        fi
-        echo "Backend started with PID $(cat /tmp/backend-e2e.pid)"
+      uses: ${{ inputs.e2e-tests-path }}/.github/actions/start-backend
```

actionlint (1.7.8)

context "inputs" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)